verinfast/cloud/aws.py

```diff
@@ -5,7 +5,6 @@
 from utils.utils import debugLog
 
 def runAws (targeted_account, start, end, path_to_output):
-
```

Comment:
This file needs to be moved on my machine, or it breaks the import of things in the aws path.